lib/evss/claims_service.rb

```diff
@@ -1,20 +1,19 @@
 module EVSS
```

Comment:
I think now would be a good time to split this up into separate files under an `evss` folder, something like `base`, `claims`, and `documents`